app/src/main/java/org/merakilearn/FileChooserActivity.kt

```diff
@@ -96,7 +105,51 @@ class FileChooserActivity : AppCompatActivity() {
                 }
             }
         }
+    }
 
+    private fun checkAndRequestPermissions() {
```

Comment:
Where we are using this permission @aayushitiwari03

Reply:
@surajsahani We are using this function inside on create method and this necessary.